draft-ietf-anima-brski-prm.md

```diff
@@ -2054,103 +2095,109 @@ Within the server log the registrar SHOULD capture this telemetry information.
 This is out of scope for this specification.
 
-{{stat_req}} below shows an example for querying pledge-status using status-type bootstrap.
+{{stat_req_data}} below shows an example for unsigned Status Query data in JSON syntax using status-type `bootstrap`.
 
 ~~~~
-# The Registrar-Agent request of "pledge-status" in general JWS
-  serialization syntax
-{
-  "payload": BASE64URL(UTF8(status-request)),
-  "signatures": [
-    {
-      "protected": BASE64URL(UTF8(JWS Protected Header)),
-      "signature": BASE64URL(JWS Signature)
-    }
-  ]
-}
-
-# Example: Decoded payload "status-request" representation
-  in JSON syntax
 {
   "version": 1,
   "created-on": "2022-08-12T02:37:39.235Z",
   "serial-number": "pledge-callee4711",
   "status-type": "bootstrap"
 }
+~~~~
+{: #stat_req_data title="Example of unsigned Status Query data in JSON syntax using status-type bootstrap for the Status Query artifact" artwork-align="left"}
 
-# Example: Decoded "JWS Protected Header" representation
-  in JSON syntax
+The Status Query data MUST be signed by the Registrar-Agent using its private key corresponding to the EE (RegAgt) certificate.
+When using a JWS signature, the Status Query artifact looks as shown in {{stat_req}} and the Content-Type response header MUST be set to `application/jose+json`:
+
+~~~~
 {
-  "alg": "ES256",
-  "x5c": [
-    "base64encodedvalue==",
-    "base64encodedvalue=="
+  "payload": BASE64URL(UTF8(status-query)),
+  "signatures": [
+    {
+      "protected": BASE64URL(UTF8(JWS Protected Header)),
+      "signature": BASE64URL(JWS Signature)
+    }
   ]
 }
 ~~~~
-{: #stat_req title='Example of Registrar-Agent request of pledge-status using status-type bootstrap' artwork-align="left"}
+{: #stat_req title="Status Query Representation in General JWS JSON Serialization Syntax" artwork-align="left"}
+
+For details on `JWS Protected Header` and `JWS Signature` see {{!I-D.ietf-anima-jws-voucher}} or {{!RFC7515}}.
+
 
+### Response Artifact: Pledge Status (pStatus)
 
-### Response Artifact: Pledge Status Response (pStatus-Resp)
+When the pledge receives a Status Query with status-type `bootstrap` it SHALL respond with previously collected telemetry information (see {{vstatus}} and {{estatus}}) in a single Pledge Status artifact.
 
-If the pledge receives the pledge-status request with status-type "bootstrap" it SHALL react with a status response message based on previously collected telemetry information (see {{vstatus}} and {{estatus}}) in a single status-response artifact.
 
-The pledge-status response Content-Type header is `application/jose+json`.
+The pledge-status response message is signed with IDevID or LDevID, depending on bootstrapping state of the pledge.
 
-The following CDDL explains the structure of the format for the status response, which is:
+The following CDDL defines the structure of the Pledge Status (pStatus) data:
 
 ~~~~
 <CODE BEGINS>
-  status-response = {
+  pledgestatus = {
```

Comment:
Closer to "enrollstatus-post" etc. in BRSKI.

Reply:
Done